pkg/orchestration/wiring/wiringutil/knownshapes/bindable_environment_variables.go

```diff
@@ -19,7 +19,8 @@ type BindableEnvironmentVariables struct {
 // +k8s:deepcopy-gen=true
 type BindableEnvironmentVariablesData struct {
 	wiringplugin.BindableShapeStruct `json:",inline"`
-	Prefix                           string `json:"prefix,omitempty"`
+	Prefix                           string            `json:"prefix,omitempty"`
+	Vars                             map[string]string `json:"vars,omitempty"`
```

Comment:
I can't see any example of that being initialized, so I assume it is not tested at all.

Add tests / examples.

Reply:
Not at this moment - there's no shapes that are producing this structure. I've tested it in my next PR - it's fine if these are unused because the code path is unused.